.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
+
+## PR Checklist
+
+-   [ ] Addresses an existing open issue: fixes #000
+-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypescript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
+-   [ ] Steps in [CONTRIBUTING.md](https://github.com/LearningTypescript/projects/blob/main/CONTRIBUTING.md) were taken
```

Comment:
Fun fact: you don't have to put these files in a `.github` directory. I just like it to keep them organized and out of the way.

```suggestion
-   [ ] Steps in [CONTRIBUTING.md](https://github.com/LearningTypeScript/projects/blob/main/.github/CONTRIBUTING.md) were taken
```